Src/Celbody/Ariel/CMakeLists.txt

```diff
@@ -28,6 +28,7 @@ set_target_properties(${CELBODY}
 	FOLDER Celbody
 )
 
+if(NOT CMAKE_CL_64) #remove me when this celbody has source code that can be built
```

Comment:
We have `BUILD64` macro in root `CMakeLists.txt` - worth using that for consistency